tests/labhub_test.py

```diff
@@ -191,20 +200,26 @@ def test_unassign_cmd(self):
         mock_iss.assignees = (None, )
         mock_iss.unassign = MagicMock()
 
-        testbot.assertCommand('!unassign https://github.com/coala/name/issues/23',
+        # Start ignoring InvalidLinkBear
+        testbot.assertCommand('!unassign https://github.com/coala/name/'
```

Comment:
move strings to next line so you dont need to split a URL.  that makes it quite hard to read and understand the test procedure/intention.